app/react-native-server/readme.md

```diff
@@ -0,0 +1,13 @@
+# Storybook Server for React Native
+Additional package for @storybook/react-native to support server.
```

Comment:
=> `to support a web server and browser UI that runs alongside the RN on-device UI`